client/src/components/Main.jsx

```diff
@@ -1,7 +1,16 @@
 import "./Main.css";
 import KanjiCard from "./KanjiCard";
+import { Fireworks } from "fireworks/lib/react";
```

Comment:
Is this path correct? I couldn't find this file in this repo.

Reply:
sorry you will need to do an npm install for this library. 
`npm install fireworks --save`

Reply:
I see! Let me try again!

Reply:
Sorry, since I reverted this PR, can you make a PR again? I'm sorry!